standard-integration/app/src/main/java/io/mimi/example/android/MyApp.kt

```diff
@@ -12,22 +14,28 @@ class MyApp : Application() {
         initializeMimiCore(this)
     }
 
-    private fun enableMimiSDKLogs() {
-        io.mimi.sdk.core.util.Log::class.java.getDeclaredField("enabled")
-                .apply {
-                    isAccessible = true
-                    setBoolean(io.mimi.sdk.core.util.Log(), true)
-                }
-    }
-
     private fun initializeMimiCore(context: Context) {
-        // Uncomment to enable debug logs for MSDK
-        // enableMimiSDKLogs()
+        enableMimiSDKLogs()
         MimiCore.start(
                 context = context,
                 configuration = MimiConfiguration(),
                 clientId = BuildConfig.MY_CLIENT_ID,
                 clientSecret = BuildConfig.MY_CLIENT_SECRET
         )
+        // This is an optional step which is only applicable if you are using the Mimi SDK to
+        // provide PTT hearing test functionality for known headphone models.
+        setUpHeadphoneIdentifierProvider()
+    }
+
+    private fun setUpHeadphoneIdentifierProvider() {
+        MimiCore.testsController.connectedHeadphoneProvider = MimiConnectedHeadphoneProvider {
+            HeadphoneIdentifierStore.instance.headphoneModelId?.let { id ->
+                MimiHeadphoneIdentifier(id)
+            }
+        }
+    }
+
+    private fun enableMimiSDKLogs() {
+        io.mimi.sdk.core.util.Log.isEnabled = BuildConfig.DEBUG
```

Comment:
Not related to this PR, but it made me wonder if we should rename this `Log` class to something else to avoid clashing with the Android one. Perhaps `MimiLog`?

Reply:
Yes, this is something that I've been considering because I often import the wrong one.

Reply:
Let's follow up on this in the future, then :)